src/ProcessGroupCCL.cpp

```diff
@@ -66,15 +66,15 @@ std::map<ReduceOp, ccl::reduction> cclOps =
 
 // Type mapping
 #if CCL_MAJOR_VERSION == 0 && CCL_MINOR_VERSION < 6
-std::map<at::ScalarType, ccl::data_type> cclDatatypes =
-{
-    {at::kByte, ccl::data_type::dt_char},
-    {at::kChar, ccl::data_type::dt_char},
-    {at::kDouble, ccl::data_type::dt_double},
-    {at::kBFloat16, ccl::data_type::dt_bfp16},
-    {at::kFloat, ccl::data_type::dt_float},
-    {at::kInt, ccl::data_type::dt_int},
-    {at::kLong, ccl::data_type::dt_int64}
+std::map<at::ScalarType, ccl::datatype> cclDatatypes =
```

Comment:
the check above already should cover difference in API
#if CCL_MAJOR_VERSION == 0 && CCL_MINOR_VERSION < 6